h2o-docs/src/product/data-science/algo-params/distribution.rst

```diff
@@ -80,6 +80,50 @@ Some examples of response distributions are provided below.
+|                   | - :math:`-2w*(y*f-exp(f))`                                                                |
++-------------------+-------------------------------------------------------------------------------------------+
+| Gamma             | - With :math:`f=link(f)`                                                                  |
+|                   | - :math:`2w*(y*exp(-f)+f)`                                                                |
++-------------------+-------------------------------------------------------------------------------------------+
+| Laplace           | :math:`w*\mid{y-f}\mid`                                                                   |
++-------------------+-------------------------------------------------------------------------------------------+
+| Quantile          | - :math:`y>f: w*QuantileAlpha*(y-f)`                                                      |
+|                   | - :math:`y<=f: w*1-QuantileAlpha*(f-y)`                                                   |
++-------------------+-------------------------------------------------------------------------------------------+
+| Huber             | - :math:`\mid{y-f}\mid <= \text{Huber_Delta}`:                                            |
+|                   | - :math:`w*(y-f)^2` (Equivalent to wMSE)                                                  |
+|                   | - :math:`y<=f`:                                                                           |
+|                   | - :math:`2w*(\mid{y-f}\mid - \text{Huber_Delta})*\text{Huber_Delta}` (Equivalent to wMAE) |
++-------------------+-------------------------------------------------------------------------------------------+
+| Modified_Huber    | - with :math:`yf=(2y-1)*f`                                                                |
+|                   | - :math:`yf<-1`:                                                                          |
+|                   | - :math:`-4w*(yf)`                                                                        |
+|                   | - :math:`yf>1` : :math:`0`                                                                |
+|                   | - else: :math:`w*(yf)^2`                                                                  |
++-------------------+-------------------------------------------------------------------------------------------+
+| Tweedie           | with :math:`f=\text{link}(f)`, requires :math:`1>\text{_tweediePower}<2`:                 |
+|                   | :math:`2w*y^{(2-\text{Tweedie_Power})/((1-\text{Tweedie_Power})*(2-\text{Tweedie_Power})) |
+|                   | -y*\text{exp}(f*(1-\text{Tweedie_Power}))/(1-\text{Tweedie_Power})                        |
+|                   | +\text{exp}(f*(2-\text{Tweedie_Power}))/(2-\text{Tweedie_Power})}`                        |
++-------------------+-------------------------------------------------------------------------------------------+
+
```

Comment:
We should also explain what `y`, `f,` and `w` mean.

y = true response
f = predicted response
w = weight

With distributions also link functions are connected. We should also mention this link functions and link inverse functions: 

- Identity: Gaussian, Huber, Laplace, Quantile

link(f) = f
linkInversion(f) = f

- Log: Multinomial, Poisson, Gamma, Tweede

link(f) = log(f)
linkInversion(f) = exp(f)

- Logit: Bernoulli, Quasibinomial, Modified Huber

link(f) = log(f / (1 - f)
linkInv(f) = 1 / (1 + exp(-f))

There are other link functions that are not associated with concrete distributions but can be used with a custom distribution: 

- Inverse
 
link(f) = 1/f (where f = f < 0 ? min(-1e-5, f) : max(-1e-5, f))
linkInversion(f) = link(f)

- Ologit

link(f) = log(f / (1 - f)
linkInversion(f) = 1 / (1 + exp(-f))

- Ologlog

link(f) = log(-1 * log(1-f) );
linkInversion(f) =1 - exp(-1 * exp(f))

- Oprobit

link(f) = normalDistribution.inverseCumulativeProbability(f)
linkInversion(f) = normalDistribution.cumulativeProbability(f)

@hannah-tillman, let me know if something is not clear to you. :)